tests/foreman/destructive/test_clone.py

```diff
@@ -0,0 +1,117 @@
+
+:Assignee: lpramuk
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+import requests
+
+from robottelo import constants
+from robottelo.config import settings
+from robottelo.hosts import Satellite
+
+SSH_PASS = settings.server.ssh_password
+pytestmark = pytest.mark.destructive
+
+
+@pytest.mark.parametrize("sat_ready_rhel", [8], indirect=True)
+@pytest.mark.parametrize('backup_type', ['online', 'offline'])
+@pytest.mark.parametrize('skip_pulp_content', [True, False], ids=['skip_pulp', 'no_skip_pulp'])
+def test_positive_clone_backup(target_sat, sat_ready_rhel, backup_type, skip_pulp_content):
+    """Make an online/offline backup with/without pulp data of Satellite and clone it (restore it).
+
+    :id: 5b9182d5-6789-4d2c-bcc3-6641b96ab277
+
+    :steps:
+        1. Create an online/offline backup with/without pulp data
+        2. Copy it to target rhel
+        3. Install satellite-clone
+        4. Run satellite clone on target rhel
+
+    :expectedresult:
+        1. Satellite-clone ran successfully
+
+    :parametrized: yes
+    """
+    rhel_version = sat_ready_rhel._v_major
+
+    # SATELLITE PART - SOURCE SERVER
+    # Enabling and starting services
+    assert target_sat.cli.Service.enable().status == 0
+    assert target_sat.cli.Service.start().status == 0
+
+    # Doing backup
+    backup_result = target_sat.cli.Backup.run_backup(
+        backup_dir='/var/backup',
+        backup_type=backup_type,
+        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': skip_pulp_content},
+    )
+    assert backup_result.status == 0
+    sat_backup_dir = backup_result.stdout.strip().split()[-2]
+
+    if skip_pulp_content:
+        # Copying satellite pulp data to target RHEL
+        assert sat_ready_rhel.execute('mkdir -p /var/lib/pulp').status == 0
+        assert (
+            target_sat.execute(
+                f'''sshpass -p "{SSH_PASS}" scp -o StrictHostKeyChecking=no \
+                -r /var/lib/pulp root@{sat_ready_rhel.hostname}:/var/lib/pulp/pulp'''
+            ).status
+            == 0
+        )
+    # Copying satellite backup to target RHEL
+    assert (
+        target_sat.execute(
+            f'''sshpass -p "{SSH_PASS}" scp -o StrictHostKeyChecking=no \
+            -r {sat_backup_dir} root@{sat_ready_rhel.hostname}:/backup/'''
+        ).status
+        == 0
+    )
+    assert target_sat.cli.Service.stop().status == 0
+    assert target_sat.cli.Service.disable().status == 0
+
+    # RHEL PART - TARGET SERVER
+    assert sat_ready_rhel.execute('ls /backup/config_files.tar.gz').status == 0
+    # Registering to cdn
+    sat_ready_rhel.register_to_cdn()
+    # Disabling repositories
+    assert sat_ready_rhel.execute('subscription-manager repos --disable=*').status == 0
+    # Getting satellite maintenace repo
+    res = requests.get(
+        f'{settings.repos.ohsnap_repo_host}/api/releases/{target_sat.version}/'
+        f'el{rhel_version}/satellite/repo_file'
```

Comment:
Without snaps this won't work !

These URLs:
```
http://ohsnap/api/releases/6.11.0/el7/satellite/repo_file
http://ohsnap/api/releases/6.11.1/el7/satellite/repo_file
http://ohsnap/api/releases/6.11.2/el7/satellite/repo_file
http://ohsnap/api/releases/6.11.3/el7/satellite/repo_file
```
returns the same repofile pointing to latest 6.11.z  
```
http://dogfood/pulp/repos/Sat6-CI/QA/Satellite_6_11_with_RHEL7_Server/custom/Satellite_6_11_Composes/Satellite_6_11_RHEL7/
```

Reply:
I would be for using `settings.repos.satmaintenace_repo` and you can then replace `s/RHEL7/RHEL8/` to get URL for other RHEL version

Reply:
Or make developers to fix this behavior

Reply:
I will resolve this one as users are now [able to specify the snap number](https://github.com/SatelliteQE/robottelo/pull/10079/files#diff-8c51f6ed7f99859d1efd6f74f29cc8e05e54bca2389c18e7e52c0203c1cbee6bR43) while requesting the repofile from Ohsnap.

See satellite/ohsnap/-/issues/30 on our internal GitLab.